### DISCUSSION TITLE
Unable to Consume ECML Content

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#730] [General Discussion]
Hello Team,

I am encountering an issue where ECML content cannot be consumed in release 5.0.

cc: @swayangjit

**Comment by swayangjit** [comment]
@siddhishinde0723 Please take [this patch](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/commit/0b2954e06fedaaa65a90a8e64e69e69451ba36d9) as reference. This is a work around for the  ecml fix.

**Reply by siddhishinde0723** [reply]
@swayangjit As a workaround, the content will get downloaded in local storage. So is there any other alternative so we dont download the content but can use a different content type?

@vinukumar-vs

**Comment by sayali2523** [comment]
@swayangjit I am encountering an issue where ECML content when targer SDK 34 on Android 14 
is there is any update? for SDK 33 it work using download content

**Reply by HarishGangula** [reply]
@sayali2523 It works with the downloaded content.

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
